src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,11 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    public static String getRandomColor() {
+
+        int index = new Random().nextInt(Colors.values().length);
+        return String.valueOf(Colors.values()[index]);
     }
```

Comment:
three checklist issues here, make sure to revisit it
https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/lottery